From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: krlvm <51774833+krlvm@users.noreply.github.com>
Date: Mon, 4 Jul 2022 16:14:37 +0300
Subject: Follow only system dark mode

Follow only system dark mode preference when theme is set to system default

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
Change-Id: I0fd68270a60dda4bc2b6af425182add61320d610
---
 .../browser/night_mode/GlobalNightModeStateController.java      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/ui/android/night_mode/java/src/org/chromium/chrome/browser/night_mode/GlobalNightModeStateController.java b/chrome/browser/ui/android/night_mode/java/src/org/chromium/chrome/browser/night_mode/GlobalNightModeStateController.java
--- a/chrome/browser/ui/android/night_mode/java/src/org/chromium/chrome/browser/night_mode/GlobalNightModeStateController.java
+++ b/chrome/browser/ui/android/night_mode/java/src/org/chromium/chrome/browser/night_mode/GlobalNightModeStateController.java
@@ -139,7 +139,7 @@ class GlobalNightModeStateController
     }
 
     private void updateNightMode() {
-        boolean powerSaveModeOn = mPowerSaveModeMonitor.powerSavingIsOn();
+        boolean powerSaveModeOn = false;
         final int theme = NightModeUtils.getThemeSetting();
         final boolean newNightModeOn =
                 theme == ThemeType.SYSTEM_DEFAULT
-- 
2.39.2

